README.md

```diff
@@ -124,7 +124,7 @@ by adding `elixir_analyzer` to your list of dependencies in `mix.exs`:
 ```elixir
 def deps do
   [
-    {:ex_mentor, "~> 0.1.0"}
+    {:elixir_analyzer, "~> 0.1.0"}
```

Comment:
I think this whole section can get removed, as its unlikely that the analyzer gets published on hex.